src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const i in actions) {
+    const action = actions[i];
+
+    if (!action.type) {
```

Comment:
2. [checklist] Use switch statement if you have limited amount of conditions.
Don't forget default case for error handling.